filetypes/file.go

```diff
@@ -4,11 +4,11 @@ import (
 	"io"
 	"os"
 
+	"github.com/h2non/filetype"
```

Comment:
I had to replace `gopkg.in/h2non/filetype.v1` by `github.com/h2non/filetype` to work-around the following error:

```
# go get -u gopkg.in/h2non/filetype.v1                                                                                               
go: gopkg.in/h2non/filetype.v1@v1.0.8: go.mod has non-....v1 module path "github.com/h2non/filetype" at revision v1.0.8                                                                 
go get: error loading module requirements                                                                                                                                               
```

cf. https://stackoverflow.com/a/55433888/316805